include/berry.h

```diff
@@ -1,6 +1,8 @@
 #ifndef GUARD_BERRY_H
 #define GUARD_BERRY_H
 
+#define NUM_BERRY_STAGES 4
```

Comment:
This should be moved to `include/constants/berry.h` and clarified, as berry trees have 6 stages of growth

Reply:
Done.